scratch/evaluate_gaming_traffic.cc

```diff
@@ -117,46 +111,48 @@ main (int argc, char *argv[])
 
   Ipv4InterfaceContainer interfaces = address.Assign (devices);
 
-  GamingStreamingServerHelper gamingStreamingHelper (gameId, interfaces.GetAddress (0), 9);
+  GamingStreamingServerHelper gamingStreamingHelper (game, interfaces.GetAddress (0), 9);
 
   ApplicationContainer serverApps = gamingStreamingHelper.Install (nodes.Get (1));
   gamingServer = StaticCast<GamingStreamingServer> (serverApps.Get (0));
-  serverApps.Start (Seconds (0.1));
-  serverApps.Stop (Seconds (simulationTime + 0.1));
+  serverApps.Start (Seconds (0.01));
+  serverApps.Stop (Seconds (simulationTime));
   /** End of Streaming Server*/
 
   PacketSinkHelper sink ("ns3::UdpSocketFactory", InetSocketAddress (Ipv4Address::GetAny (), 9));
   ApplicationContainer sinkApps = sink.Install (nodes.Get (0));
   packetSink = StaticCast<PacketSink> (sinkApps.Get (0));
   sinkApps.Start (Seconds (0.0));
-  sinkApps.Stop (Seconds (simulationTime + 0.2));
-  if (throughput)
-    {
-      Simulator::Schedule (Seconds (0.1), &CalculateThroughput);
-    }
-  packetSink->TraceConnectWithoutContext ("Rx", MakeCallback (&AccumulateDelay));
+  sinkApps.Stop (Seconds (simulationTime));
 
-  if (packetStats)
+  if (csv)
     {
-      gamingServer->TraceConnectWithoutContext ("Tx", MakeCallback (&GeneratedPacketsStats));
+      AsciiTraceHelper ascii;
+      Ptr<OutputStreamWrapper> cdfResults = ascii.CreateFileStream ("cdfResults.csv");
+      *cdfResults->GetStream () << "PKT_SIZE, IAT" << std::endl;
```

Comment:
spaces should be avoided in csv format
```suggestion
      *cdfResults->GetStream () << "PKT_SIZE,IAT" << std::endl;
```